shared/profile/prove-enter-username-container.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+import React, {Component} from 'react'
+import ProveEnterUsername from './prove-enter-username'
+import {bindActionCreators} from 'redux'
+import {connect} from 'react-redux'
+import {submitUsername, cancelAddProof, updateUsername} from '../actions/profile'
+
+class ProveEnterUsernameContainer extends Component<void, any, void> {
```

Comment:
originally i didn't need this component but i do for the parseRoute function. when we revisit the meta-navigator we could remove this requirement then we could get rid of this wrapper